[2023-09-13T07:38:45.364+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.392+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.392+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:45.419+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-07 00:00:00+00:00
[2023-09-13T07:38:45.441+0000] {standard_task_runner.py:57} INFO - Started process 147 to run task
[2023-09-13T07:38:45.469+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-07T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp1k_q3k9z']
[2023-09-13T07:38:45.482+0000] {standard_task_runner.py:85} INFO - Job 37: Subtask extract_data
[2023-09-13T07:38:45.643+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:45.880+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-07T00:00:00+00:00'
[2023-09-13T07:38:46.182+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:46.204+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230907T000000, start_date=20230913T073845, end_date=20230913T073846
[2023-09-13T07:38:46.262+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:46.280+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:20.219+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [queued]>
[2023-09-15T11:31:20.239+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [queued]>
[2023-09-15T11:31:20.240+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:20.259+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-07 00:00:00+00:00
[2023-09-15T11:31:20.265+0000] {standard_task_runner.py:57} INFO - Started process 378 to run task
[2023-09-15T11:31:20.280+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-07T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp4t6uqkm7']
[2023-09-15T11:31:20.282+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask extract_data
[2023-09-15T11:31:20.377+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-07T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:20.564+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-07T00:00:00+00:00'
[2023-09-15T11:31:20.871+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:20.889+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230907T000000, start_date=20230915T113120, end_date=20230915T113120
[2023-09-15T11:31:20.943+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:20.968+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
